tox.ini

```diff
@@ -1,19 +1,13 @@
 [tox]
-envlist = py36, py37, py38, py39, py310, linting, docs
+envlist = py{38,39,310,311,312}, docs
 isolated_build = true
 
 [testenv]
 extras = testing
 commands =
-    pytest --cov={envsitepackagesdir}/barril --pyargs barril {posargs} --doctest-modules
+    pytest --cov={envsitepackagesdir}/barril --pyargs barril {posargs} --doctest-modules --color=yes
     codecov
 
-[testenv:linting]
-skip_install = True
-basepython = python3.6
-deps = pre-commit>=1.11.0
-commands = pre-commit run --all-files --show-diff-on-failure
-
```

Comment:
No more linting?

Reply:
Done automatically by `pre-commit.ci`, so this no longer needed.